src/routes/index.svelte

```diff
@@ -1,11 +1,17 @@
 <script lang="ts">
+  import { userStore } from '$lib/stores/user';
+
   import Feed from '$lib/components/Feed.svelte';
+  import PostBox from '$lib/components/PostBox.svelte';
 </script>
 
 <svelte:head>
   <title>Nexus</title>
 </svelte:head>
 
 <div class="min-w-screen grid gap-4 grid-cols-6 md:grid-cols-12">
+  {#if !$userStore.user === null}
```

Comment:
This is an irrefutable evaluation, based on the following facts:

1. Whenever you use the negation operator (`!`), you turn the right-hand value into a boolean
2. As you are comparing against `null` with a strict equality operator, neither `false` nor `true` will ever be equals to `null`
3. As a result, this will always evaluates to `false`